about.html

```diff
@@ -12,81 +13,102 @@
 </head>
 
 <body>
-    <div class="container about-page">
-        <div class="about-header-container">
-            <img src="img/lambda-black.png">
-            <nav>
-                <a href="index.html">Home</a>
-                <a href="about.html">About</a>
-                <a href="#">Products</a>
-                <a href="#">Blog</a>
-                <a href="#">Contact</a>
-            </nav>
-            
-        </div>
-        <img class="jumbo-about" src="img/jumbo-about.png">
+  <div class="container about-page">
+    <div class="about-header-container">
+      <img src="img/lambda-black.png">
+      <nav>
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+      </nav>
+
+    </div>
+    <img class="jumbo-about" src="img/jumbo-about.png">
 
     <div class="sectionAbout">
 
-   <h3>About</h3>
+      <h3>About</h3>
 
-    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to
+        corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the
+        holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
 
-  </div>
+    </div>
 
 
-  <div class="sectionStrategy"></div>
+    <div class="mainSection">
 
-    <img class="about-plan" src="img/about-plan.png" alt="strategy">
-  
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      <div class="subSection">
```

Comment:
Great job with the html and class naming here!  It's easy to understand.  I love that the class names are reusable so that the code is DRY.